amethyst_renderer/src/pass/skybox/interleaved.rs

```diff
@@ -16,6 +14,8 @@ use {
     },
     set_vertex_args, ActiveCamera, Camera, Encoder, Factory, Mesh, PosTex, Shape, VertexFormat,
 };
+use gfx::pso::buffer::ElemStride;
+use glsl_layout::{mat4, Uniform};
```

Comment:
Why did you move these?

Reply:
`cargo fmt` did.  I guess it likes them better there.

Reply:
Does it do it even if you add a newline between the imports?